CONTRIBUTING.md

```diff
@@ -120,3 +125,9 @@ While not all code submitted at this time conforms to this standard, it is the i
     - `1.f`
     - `1.L`
   - No: `1` for `f32` or `f64`
+
+## <a name="prs"></a>Pull Requests
+
+In order to make modifications to this repo, you'll have to fork the repo to your personal GitHub. There, you're free to make edits however you want. When actually creating a PR, however, try to match whole files at a time as PR reviewers are a scarce resource.
```

Comment:
```suggestion
In order to make modifications to this repo, you'll have to [fork the repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo) to your personal GitHub. There, you're free to make edits however you want. When actually creating a PR, however, try to match whole files at a time as PR reviewers are a scarce resource.
```

Reply:
Something along the lines of #530 should also be mentioned, because it's much easier than forking, compiling the codebase yourself, and then creating a PR.